AntennaTracker/GCS_MAVLink_Tracker.cpp

```diff
@@ -558,7 +560,7 @@ void GCS_MAVLINK_Tracker::send_global_position_int()
         AP_HAL::millis(),
         tracker.current_loc.lat,  // in 1E7 degrees
         tracker.current_loc.lng,  // in 1E7 degrees
-        tracker.current_loc.alt,  // millimeters above ground/sea level
+        tracker.current_loc.get_alt_cm(),  // millimeters above ground/sea level
```

Comment:
millimeters?

Reply:
Huh.  Fun.  Comment is definitely wrong, `.alt` is cm